src/platform/plugins/shared/saved_search/server/saved_objects/schema.ts

```diff
@@ -156,3 +156,18 @@ export type DiscoverSessionTab = TypeOf<typeof SCHEMA_DISCOVER_SESSION_TAB>;
 export const SCHEMA_SEARCH_MODEL_VERSION_6 = SCHEMA_SEARCH_MODEL_VERSION_5.extends({
   tabs: schema.maybe(schema.arrayOf(SCHEMA_DISCOVER_SESSION_TAB, { minSize: 1 })),
 });
+
+const { columns, grid, hideChart, isTextBasedQuery, kibanaSavedObjectMeta, rowHeight, sort } =
+  SCHEMA_SEARCH_MODEL_VERSION_6.getPropSchemas();
+
+// Mark top-level attributes (except title and description) optional, and mark tabs as required
+export const SCHEMA_SEARCH_MODEL_VERSION_7 = SCHEMA_SEARCH_MODEL_VERSION_6.extends({
+  columns: schema.maybe(columns),
```

Comment:
This only includes the properties that were not already marked as options (`schema.maybe`).